apps/kampus/app/pano/features/filter/utils.ts

```diff
@@ -0,0 +1,13 @@
+export type PanoPostFilterType = "all" | "active" | "hot" | "liked" | "my-posts";
+
+export const filters: Record<PanoPostFilterType, string> = {
+  all: "hepsi",
+  active: "en son yorumlananlar",
+  hot: "en fazla yorum almışlar",
+  liked: "beğendiklerim",
+  "my-posts": "başlıklarım",
+};
```

Comment:
not sure about this but we can figure out later. It's interesting dictionary to work with.